### PR TITLE
run tests on LDAPS

### LINKED ISSUE #348: run acceptance tests with LDAPS
currently all tests run with unencrypted LDAP, we also should run them with LDAPS

### PR DESCRIPTION
**Pull request by individual-it**
fixes  #348

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -126,6 +127,27 @@ pipeline:
...
+      matrix:
+        NEED_SERVER: true
+
+  ldaps-config-for-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/server/
+      - php occ ldap:set-config LDAPTestId ldapPort "636"
+      - php occ ldap:set-config LDAPTestId ldapHost "ldaps://ldap"
```
Comment (skshetry) [inline .drone.yml]: Setting `ldaps://` doesnot require us to set `ldapTLS`?

Comment (individual-it) [reply .drone.yml]: actually both cannot be set at the same time. I had to remove `ldapTLS=1` again

**Comment by skshetry** [comment]
@individual-it There seems to be few tests (~5) that sets host to `ldap`, which are failing.

**Comment by individual-it** [comment]
Yes, when testing failing and reestablished connections we set the host with `ldap://` and the port to `389` that obviously does not work with LDAPS, need something more clever here

File: tests/acceptance/features/bootstrap/UserLdapGeneralContext.php [hunk 2]
```diff
@@ -25,6 +25,7 @@
 use Behat\Gherkin\Node\PyStringNode;
 use Behat\Gherkin\Node\TableNode;
 use Behat\MinkExtension\Context\RawMinkContext;
+use PHPUnit\Framework\Assert;
```
Comment (skshetry) [inline tests/acceptance/features/bootstrap/UserLdapGeneralContext.php]: Do we need to import `Assert` namespace?

File: tests/acceptance/config/behat.yml [hunk 3]
```diff
@@ -8,11 +8,7 @@ default:
         - '%paths.base%/../../../../../tests/acceptance/features'
       context: &common_ldap_suite_context
         parameters:
-          ldapAdminUser: cn=admin,dc=owncloud,dc=com
           ldapAdminPassword: admin
-          ldapBaseDN: dc=owncloud,dc=com
```
Comment (skshetry) [inline tests/acceptance/config/behat.yml]: We do not use this anywhere, do we?

Comment (individual-it) [reply tests/acceptance/config/behat.yml]: that setting is needed for setting the LDAP connection see https://github.com/owncloud/user_ldap/pull/415/files#diff-1fd6e7b9542409bf830dd808e362648aL422
before we used the setting from this yml, now from the current LDAP setting see https://github.com/owncloud/user_ldap/pull/415/files#diff-1fd6e7b9542409bf830dd808e362648aR438

File: tests/acceptance/features/bootstrap/UserLdapGeneralContext.php (hunk 2 above)
Comment (individual-it) [reply tests/acceptance/features/bootstrap/UserLdapGeneralContext.php]: good catch, fixed

**Pull request merged**